code/modules/events/wisdomcow.dm

```diff
@@ -0,0 +1,16 @@
+/datum/round_event_control/wisdomcow
+	name = "Wisdom cow"
+	typepath = /datum/round_event/wisdomcow
+	max_occurrences = 1
+	weight = 20
+
+/datum/round_event/wisdomcow/announce(fake)
+	priority_announce("A wise cow has been spotted in the area. Be sure to ask her advice.", "Nanotrasen Cow Ranching Agency")
```

Comment:
shouldn't it be ask for her advice or ask her for advice

Reply:
This would be correct